### PR TITLE
Use single except catch in controllers

### LINKED ISSUE #78: Raise exception instead of returning none for database operations
**Describe the improvement you would like to see**
Currently the output of read operations from the database is returned directly. The return value might include none, which needs to be checked in the controller. This makes the controller logic more complicated. Instead the crud operations should raise some custom errors that can be handled in the controller.

**How will this change existing functionality?**
Make error handling uniform in the controller. The issue is related to #74

**How will this improvement affect the current Threat Model?**
No changes.

### PR DESCRIPTION
**Pull request by vetlek**
Closes #78 and #74

### CODE REVIEW

**Review by anetteu — CHANGES_REQUESTED** [review]
Good work!

**Review by anetteu — DISMISSED** [review]

File: backend/src/flotilla/api/missions_api.py [hunk 1]
```diff
@@ -26,24 +29,27 @@
...
 ) -> List[Mission]:
     """### Overview List all available missions on the asset in the Echo mission planner"""
     try:
         echo_response: RequestResponse = echo_requests.get_missions()
+        missions: List[Mission] = []
+        for mission in echo_response.json():
+            try:
+                missions.append(mission_deserializer(mission))
+            except EchoDeserializerException as e:
+                continue
```
[the remark this answers is not in the document]
Comment (vetlek) [reply backend/src/flotilla/api/missions_api.py:43]: Feel that the deserializing to the mission format used in the API should be independent from the EchoService, which only is used to contact Echo.

**Review by nicholasdalhaug — CHANGES_REQUESTED** [review]

File: backend/src/flotilla/api/missions_api.py [hunk 2]
```diff
@@ -26,24 +29,27 @@
...
 ) -> List[Mission]:
     """### Overview List all available missions on the asset in the Echo mission planner"""
     try:
         echo_response: RequestResponse = echo_requests.get_missions()
+        missions: List[Mission] = []
+        for mission in echo_response.json():
+            try:
+                missions.append(mission_deserializer(mission))
+            except EchoDeserializerException:
+                continue
```
[the remark this answers is not in the document]
Comment (vetlek) [reply backend/src/flotilla/api/missions_api.py]: Done

**Review by nicholasdalhaug — APPROVED** [review]
🎉

**Pull request merged**